revocation_list.go

```diff
@@ -0,0 +1,39 @@
+package jwt
+
+import (
+	"time"
+)
+
+// RevocationList is used to store a mapping of public keys to unix timestamps
+type RevocationList map[string]int64
+
+// Revoke enters a revocation by publickey and timestamp into this export
+// If there is already a revocation for this public key that is newer, it is kept.
+func (r RevocationList) Revoke(pubKey string, timestamp time.Time) {
+	newTS := timestamp.Unix()
```

Comment:
I see in other place with do an explicit UTC() call as well. Not sure if its needed but we should be consistent.

Reply:
i think the unix does it automatically, but i haven't cleaned it all up.

Reply:
Sounds good.

Reply:
Still think this is Revoke(pubkey) and there could be a RevokeAtTime(pubKey, time)